### ISSUE TITLE
progress bar  - usability

### ISSUE THREAD

**Issue by ImreSamu** [julien-noblet/download-geofabrik#26]
Some geofabrik extracts so huge 
* extreme example:    europe.osh.pbf :  File size: 33.2 GB

Sometimes it will be useful to check the current status.
```
2.06 MiB / 220.65 MiB [>-----------------------------]   0.93% 1.94 MiB/s 1m52s
```

background:
-  I am [integrating this tool](https://github.com/julien-noblet/download-geofabrik/wiki/Powered-By) - and my experience that "end-users" don't know the real data sizes.
- Sometimes the download speed is very slow, and it is better to cancel and try later.
  
if possible with a flag:   "no progress bar"
I have no preference:  https://golanglibs.com/top?q=progress+bar
Not critical, not urgent ...

**Comment by julien-noblet** [comment]
This is a great idea!
It was not so hard to add.
Can you look at [v2.2.0-beta-fix#26](https://github.com/julien-noblet/download-geofabrik/releases/tag/v2.2.0-beta-fix%2326)

**Comment by ImreSamu** [comment]
Thank you! 

```
root@f13349d176b5:/tools# ./download-geofabrik download romania  --progress
 171.13 MiB / 171.14 MiB [================================================================================================================================================] 100.00% 14.00 MiB/s2018/03/25 07:26:05 romania.osm.pbf downloaded.
 171.14 MiB / 171.14 MiB [============================================================================================================================================] 100.00% 14.00 MiB/s 12s
 0 B / 57 B [-------------------------------------------------------------------------------------------------------------------------------------------------------------------------]   0.00%2018/03/25 07:26:05 romania.osm.pbf.md5 downloaded.
 57 B / 57 B [========================================================================================================================================================] 100.00% 154.16 KiB/s 0s
2018/03/25 07:26:06 Checksum OK for romania.osm.pbf
root@f13349d176b5:/tools#
```

imho:  the progress bar for the  `.md5` file is less important  ( it is always less than 100 byte )

**Comment by julien-noblet** [comment]
* I've remove progress bar for files < 512kB
* I've also add a progress bar for ```download-geofabrik generate --progress```

Have look to [fix#26](https://github.com/julien-noblet/download-geofabrik/releases/tag/fix%2326-2)

**Comment by ImreSamu** [comment]
tested !!! -  Looks OK!

```
root@1a750d861118:/tools# ./download-geofabrik generate --progress
 393 / 392 [==========================================================================================================================================================================================] 100.26%2018/04/03 18:11:41 ./geofabrik.yml  generated.
root@1a750d861118:/tools# ./download-geofabrik version
download-geofabrik: error: expected command but got "version", try --help
root@1a750d861118:/tools# ./download-geofabrik --version
2.2.0-beta
root@1a750d861118:/tools# ./download-geofabrik download monaco
2018/04/03 18:13:18 monaco.osm.pbf downloaded.
2018/04/03 18:13:18 monaco.osm.pbf.md5 downloaded.
2018/04/03 18:13:18 Checksum OK for monaco.osm.pbf
root@1a750d861118:/tools#
```
